### DISCUSSION TITLE
Is OrientDB Schema Creation possible in multi thread environment?

### DISCUSSION THREAD

**Discussion by arpitdd123** [orientechnologies/orientdb#10231] [Q&A]
I have a requirement where I want to create more than 400 vertices and edges schema along with 15 to 20 different properties in the orientdb. 

Is it possible to create schema in orientdb on threads? Because as per "https://orientdb.com/docs/3.2.x/general/Schema.html?highlight=schema#schema", "Changes to the schema are not transactional. You must execute these commands outside of a transaction.". 

Sequential orientdb schema creation take more time, so is there any way to make schema creation fast.

Thanks

**Comment by tglman** [comment]
Hi,

Yes it is possible to run schema command in multithread, in should improve a bit the performance anyway most of the work is done behind a exclusive lock so you should not see much difference, especially with an empty database.
